Update accept_batch

itsmarcy opened the pull request:
- Returns an exit code of 0 or 1 depending on status. This is
apart of the effort to automate the whole process from a bash
script (to detect failure).
- Added a method that checks if the given TSV file exist, if not
to mark the acceptance as a failure.
- Reorganized code.
- Removed `aspera_path` as it it was not being used.

itsmarcy committed:
Update accept_batch

- Returns an exit code of 0 or 1 depending on status. This is
apart of the effort to automate the whole process from another
script.
- Added a method that checks if the given TSV file exist, if not
to mark the acceptance as a failure.
- Reorganized code.

walkerh reviewed (approved):
I note that CI checks (`pytest`) are failing, but this is due to a deprecation error (`YAMLLoadWarning`) in PyYAML, and the fact that we did not pin the version of PyYAML — by design.

I will open a new issue regarding the `YAMLLoadWarning`.

itsmarcy committed:
Readd aspera_path

- Just in case it's being used anywhere else by somethign else.
Left TODO comment
